### ISSUE TITLE
Code Coverage Trend: Too low color contrast between line and branch coverage

### ISSUE THREAD

**Issue by viceice** [jenkinsci/code-coverage-api-plugin#673]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
  Jenkins: 2.387.3
OS: Linux - 5.15.0-1034-azure
Java: 11.0.19 - Eclipse Adoptium (OpenJDK 64-Bit Server VM)
---
Office-365-Connector:4.18.0
additional-identities-plugin:1.1
analysis-model-api:11.2.0
ansicolor:1.0.2
antisamy-markup-formatter:159.v25b_c67cd35fb_
apache-httpcomponents-client-4-api:4.5.14-150.v7a_b_9d17134a_5
authentication-tokens:1.53.v1c90fd9191a_b_
authorize-project:1.5.1
aws-credentials:191.vcb_f183ce58b_9
aws-java-sdk:1.12.447-382.vda_68e2007233
aws-java-sdk-cloudformation:1.12.447-382.vda_68e2007233
aws-java-sdk-codebuild:1.12.447-382.vda_68e2007233
aws-java-sdk-ec2:1.12.447-382.vda_68e2007233
aws-java-sdk-ecr:1.12.447-382.vda_68e2007233
aws-java-sdk-ecs:1.12.447-382.vda_68e2007233
aws-java-sdk-efs:1.12.447-382.vda_68e2007233
aws-java-sdk-elasticbeanstalk:1.12.447-382.vda_68e2007233
aws-java-sdk-iam:1.12.447-382.vda_68e2007233
aws-java-sdk-kinesis:1.12.447-382.vda_68e2007233
aws-java-sdk-logs:1.12.447-382.vda_68e2007233
aws-java-sdk-minimal:1.12.447-382.vda_68e2007233
aws-java-sdk-sns:1.12.447-382.vda_68e2007233
aws-java-sdk-sqs:1.12.447-382.vda_68e2007233
aws-java-sdk-ssm:1.12.447-382.vda_68e2007233
basic-branch-build-strategies:71.vc1421f89888e
bootstrap4-api:4.6.0-5
bootstrap5-api:5.2.2-3
bouncycastle-api:2.27
branch-api:2.1071.v1a_188a_562481
build-token-root:151.va_e52fe3215fc
caffeine-api:3.1.6-115.vb_8b_b_328e59d8
checks-api:2.0.0
cloudbees-folder:6.815.v0dd5a_cb_40e0e
code-coverage-api:4.5.0
command-launcher:100.v2f6722292ee8
commons-httpclient3-api:3.1-3
commons-lang3-api:3.12.0-36.vd97de6465d5b_
commons-text-api:1.10.0-36.vc008c8fcda_7b_
config-file-provider:938.ve2b_8a_591c596
credentials:1254.vb_96f366e7b_a_d
credentials-binding:604.vb_64480b_c56ca_
data-tables-api:1.13.3-3
display-url-api:2.3.7
docker-commons:419.v8e3cd84ef49c
docker-java-api:3.2.13-68.va_875df25a_b_45
docker-plugin:1.3.0
docker-workflow:563.vd5d2e5c4007f
durable-task:506.v1b_3e14b_6f5da_
echarts-api:5.4.0-3
email-ext:2.96
embeddable-build-status:369.vb_a_68a_575a_b_11
external-monitor-job:203.v683c09d993b_9
favorite:2.4.1
font-awesome-api:6.3.0-2
forensics-api:2.1.0
git:5.0.1
git-client:4.2.0
git-forensics:2.0.0
git-parameter:0.9.18
gitea:1.4.5
gitea-checks:449.v0672c320ee47
github:1.37.0
github-api:1.303-417.ve35d9dd78549
github-branch-source:1703.vd5a_2b_29c6cdc
google-oauth-plugin:1.0.8
gravatar:2.2
handy-uri-templates-2-api:2.1.8-22.v77d5b_75e6953
htmlpublisher:1.31
instance-identity:142.v04572ca_5b_265
ionicons-api:45.vf54fca_5d2154
jackson2-api:2.15.0-334.v317a_165f9b_7c
jakarta-activation-api:2.0.1-3
jakarta-mail-api:2.0.1-3
javax-activation-api:1.2.0-6
javax-mail-api:1.6.2-9
jaxb:2.3.8-1
jdk-tool:66.vd8fa_64ee91b_d
jjwt-api:0.11.5-77.v646c772fddb_0
jobcacher:378.v5e7c64638d63
jquery:1.12.4-1
jquery3-api:3.6.4-1
jsch:0.2.8-65.v052c39de79b_2
junit:1202.v79a_986785076
kubernetes:3923.v294a_d4250b_91
kubernetes-client-api:6.4.1-215.v2ed17097a_8e9
kubernetes-credentials:0.10.0
ldap:673.v034ec70ec2b_b_
lockable-resources:1156.v5e9f897ece02
mailer:448.v5b_97805e3767
matrix-auth:3.1.7
matrix-project:789.v57a_725b_63c79
metrics:4.2.13-420.vea_2f17932dd6
mina-sshd-api-common:2.9.2-62.v199162f0a_2f8
mina-sshd-api-core:2.9.2-62.v199162f0a_2f8
nunit:380.va_067eddf00da_
oauth-credentials:0.645.ve666a_c332668
okhttp-api:4.10.0-132.v7a_7b_91cef39c
pipeline-build-step:491.v1fec530da_858
pipeline-graph-analysis:202.va_d268e64deb_3
pipeline-graph-view:190.vb_6fdf3c1ccfa
pipeline-groovy-lib:656.va_a_ceeb_6ffb_f7
pipeline-input-step:468.va_5db_051498a_4
pipeline-milestone-step:111.v449306f708b_7
pipeline-model-api:2.2131.vb_9788088fdb_5
pipeline-model-definition:2.2131.vb_9788088fdb_5
pipeline-model-extensions:2.2131.vb_9788088fdb_5
pipeline-multibranch-defaults:2.1
pipeline-rest-api:2.32
pipeline-stage-step:305.ve96d0205c1c6
pipeline-stage-tags-metadata:2.2131.vb_9788088fdb_5
pipeline-stage-view:2.32
pipeline-utility-steps:2.15.2
plain-credentials:143.v1b_df8b_d3b_e48
plugin-util-api:3.2.0
popper-api:1.16.1-3
popper2-api:2.11.6-2
prism-api:1.29.0-4
pubsub-light:1.17
saml:4.403.v423b_3195a_9ec
scm-api:667.v8b_6e07cdc7f2
scoring-load-balancer:59.vf791549fa_989
script-security:1244.ve463715a_f89c
sidebar-link:2.2.1
simple-theme-plugin:160.vb_76454b_67900
snakeyaml-api:1.33-95.va_b_a_e3e47b_fa_4
sse-gateway:1.26
ssh-agent:333.v878b_53c89511
ssh-credentials:305.v8f4381501156
ssh-slaves:2.877.v365f5eb_a_b_eec
sshd:3.275.v9e17c10f2571
structs:324.va_f5d6774f3a_d
timestamper:1.25
token-macro:359.vb_cde11682e0c
trilead-api:2.84.v72119de229b_7
variant:59.vf075fe829ccb
versioncolumn:145.va_e3ca_f8a_a_d23
warnings-ng:10.1.0
workflow-aggregator:596.v8c21c963d92d
workflow-api:1213.v646def1087f9
workflow-basic-steps:1017.vb_45b_302f0cea_
workflow-cps:3659.v582dc37621d8
workflow-durable-task-step:1246.v5524618ea_097
workflow-job:1292.v27d8cc3e2602
workflow-multibranch:746.v05814d19c001
workflow-scm-step:408.v7d5b_135a_b_d49
workflow-step-api:639.v6eca_cd8c04a_a_
workflow-support:839.v35e2736cfd5c

  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

linux / windows, but doesn't matter

### Reproduction steps

1. record coverage
2. view trend chart

### Expected Results

Have more color contrast between branch and line coverage

### Actual Results

Branch coverage is nearly not visible:
![image](https://github.com/jenkinsci/code-coverage-api-plugin/assets/1798109/eb8c67e9-0fd2-4e2f-b411-dd12e7a492c9)

### Anything else?

One of the chart colors should be different or there should be more color space beween the two green colors

**Comment by uhafner** [comment]
@viceice Can you have a look at the screenshots in https://github.com/jenkinsci/code-coverage-api-plugin/pull/676? The light theme looks good, the dark theme seems could be improved, though... This should be fixed in the dark theme plugin then...

**Comment by viceice** [comment]
looks good to me 👍
